src/v0/destinations/indicative/transform.js

```diff
@@ -101,6 +102,11 @@ const handleProperties = (properties) => {
 
 const responseBuilderSimple = (message, category, destination) => {
   const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
+  // Passing user agent info for identify, track, screen and page calls
+  if (category.name !== CONFIG_CATEGORIES.ALIAS.name) {
```

Comment:
why this condition ?

Reply:
Indicative Alias call doesn't support properties